### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by praneesha**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,26 +13,26 @@ It allows you to access the [Firebase Cloud Messaging (FCM)](https://firebase.go
...
 
-1. Get a clone or download the source from [github](https://github.com/wso2-extensions/esb-connector-googlefirebase).
-2. Run the following Maven command from the `esb-connector-googlefirebase` directory: `mvn clean install`.
-3. Google Firebase connector zip will be created under `esb-connector-googlefirebase/target` directory.
+1. Get a clone or download the source from [GitHub](https://github.com/wso2-extensions/esb-connector-googlefirebase).
+2. Navigate to the the `esb-connector-googlefirebase` directory and execute the following Maven command: `mvn clean install`.
+This creates a ZIP file of the Google Firebase connector under the `esb-connector-googlefirebase/target` directory.
 
-## How You Can Contribute
+## Contributing to WSO2 Extensions
```
Comment (Biruntha) [inline README.md:37]: Shall we keep as it is. bcz in other extensions readme we are keeping " "How You Can Contribute"".

**Review by keerthu — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Google Firebase EI Connector
 
 The Google Firebase [connector](https://docs.wso2.com/display/EI640/Working+with+Connectors) allows you to integrate your own backend services with Firebase Cloud Messaging (FCM).
-It handles authenticating with Firebase servers while facilitating sending messages and managing topic subscriptions. Using FCM, you can notify a client app that new email or other data is available to sync.
+It handles authenticating with Firebase servers while facilitating sending messages and managing topic subscriptions. Using FCM, you can notify a client app that a new email or other data are available to sync.
```
Comment (praneesha) [inline README.md:4]:
```suggestion
It handles authenticating using Firebase servers while facilitating sending messages and managing topic subscriptions. Using FCM, you can notify a client app that a new email or other data are available to sync.
```

File: README.md [hunk 3]
```diff
@@ -13,26 +13,29 @@ It allows you to access the [Firebase Cloud Messaging (FCM)](https://firebase.go
...
 1. Download the connector from [WSO2 Store](https://store.wso2.com/store/assets/esbconnector/details/64a497ef-ca4c-4210-9215-da7946888a45) by clicking the Download Connector button.
 2. Then you can follow this [Documentation](https://docs.wso2.com/display/EI640/Working+with+Connectors+via+the+Management+Console) to add and enable the connector via the Management Console in your EI instance.
 3. For more information on using connectors and their operations in your EI configurations, see [Using a Connector](https://docs.wso2.com/display/EI640/Using+a+Connector).
 4. If you want to work with connectors via EI tooling, see [Working with Connectors via Tooling](https://docs.wso2.com/display/EI640/Working+with+Connectors+via+Tooling).
 
+
 ###### Configuring the connector operations
 
-To get started with google firebase connector and their operations, see [Configuring Google Firebase Operations](docs/config.md).
+To get started with Google Firebase connector and their operations, see [Configuring Google Firebase Operations](docs/config.md).
```
Comment (praneesha) [inline README.md:27]:
```suggestion
To get started with the Google Firebase connector and their operations, see [Configuring Google Firebase Operations](docs/config.md).
```

File: README.md [hunk 4]
```diff
@@ -13,26 +13,29 @@ It allows you to access the [Firebase Cloud Messaging (FCM)](https://firebase.go
...
+## Building from the Source
 
-If you want to build Google Firebase connector from the source code:
+If you want to build the Google Firebase connector from the source code:
 
-1. Get a clone or download the source from [github](https://github.com/wso2-extensions/esb-connector-googlefirebase).
-2. Run the following Maven command from the `esb-connector-googlefirebase` directory: `mvn clean install`.
-3. Google Firebase connector zip will be created under `esb-connector-googlefirebase/target` directory.
+1. Get a clone or download the source from [GitHub](https://github.com/wso2-extensions/esb-connector-googlefirebase).
+2. Navigate to the the `esb-connector-googlefirebase` directory and execute the following Maven command: `mvn clean install`.
```
Comment (praneesha) [inline README.md:34]:
```suggestion
2. Navigate to the the `esb-connector-googlefirebase` directory, and execute the following Maven command: `mvn clean install`.
```

File: README.md [hunk 5]
```diff
@@ -13,26 +13,29 @@ It allows you to access the [Firebase Cloud Messaging (FCM)](https://firebase.go
...
+This creates a ZIP file of the Google Firebase connector under the `esb-connector-googlefirebase/target` directory.
```
Comment (praneesha) [inline README.md:35]:
```suggestion
This creates a ZIP file of the Google Firebase connector inside the `esb-connector-googlefirebase/target` directory.
```

**Pull request closed without merging**